Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -63,7 +64,7 @@
 validationLiteTracking.replace(globalValidation,globalValidationLiteTracking)
 validationLiteTracking.remove(condDataValidation)
 
-validationMiniAOD = cms.Sequence(type0PFMEtCorrectionPFCandToVertexAssociationForValidationMiniAOD * JetValidationMiniAOD * METValidationMiniAOD)
+validationMiniAOD = cms.Sequence(type0PFMEtCorrectionPFCandToVertexAssociationForValidationMiniAOD * JetValidationMiniAOD * METValidationMiniAOD * tauValidationSequenceMiniAOD)
```

Comment:
in order to keep uniformity, couldn't the word Sequence be removed from tauValidationSequenceMiniAOD?

Reply:
@jfernan2 
Unfortunately this change will be in conflict with:
https://github.com/cms-sw/cmssw/blob/d15737307b0e7b710c075f0e9303e9aeee169470/Validation/RecoTau/python/RecoTauValidation_cff.py#L3-L4